app/helpers/bento_helper.rb

```diff
@@ -0,0 +1,9 @@
+module BentoHelper
+  def box_enabled?(box)
+    session[:boxes].include?(box)
+  end
+
+  def box_width
+    (12 / session[:boxes].count).to_i
```

Comment:
I'll defer to @frrrances here, but I want to say this might be a candidate for using flex-box for layout - rather than having to calculate the number of panels in the display and set widths accordingly, we should be able to have widths auto-calculated.

This isn't to say I object to this approach, especially for now - just that this might need to be revisited when the "real" UI design gets implemented.